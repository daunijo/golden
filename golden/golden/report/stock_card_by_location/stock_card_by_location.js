// Copyright (c) 2016, RSS and contributors
// For license information, please see license.txt
/* eslint-disable */

frappe.query_reports["Stock Card by Location"] = {
	"filters": [
		{
			"fieldname":"company",
			"label": __("Company"),
			"fieldtype": "Link",
			"options": "Company",
			"default": frappe.defaults.get_user_default("Company"),
			"reqd": 1
		},
		{
			"fieldname":"from_date",
			"label": __("From Date"),
			"fieldtype": "Date",
			"default": frappe.datetime.add_months(frappe.datetime.get_today(), -1),
			"reqd": 1
		},
		{
			"fieldname":"to_date",
			"label": __("To Date"),
			"fieldtype": "Date",
			"default": frappe.datetime.get_today(),
			"reqd": 1
		},
		{
			"fieldname":"item_code",
			"label": __("Item"),
			"fieldtype": "Link",
			"options": "Item",
			"reqd": 1
		},
		{
			"fieldname":"location",
			"label": __("Location"),
			"fieldtype": "Link",
			"options": "Warehouse",
			"reqd": 1,
			"get_query": function() {
				return{
					filters: {
						'is_group': 0,
						'type': "Location"
					}
				};
			}
		},
	]
}
