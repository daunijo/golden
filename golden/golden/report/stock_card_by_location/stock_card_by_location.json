{
 "add_total_row": 0, 
 "apply_user_permissions": 1, 
 "creation": "2018-08-05 00:36:22.395626", 
 "disabled": 0, 
 "docstatus": 0, 
 "doctype": "Report", 
 "idx": 0, 
 "is_standard": "Yes", 
 "letter_head": "Standard", 
 "modified": "2018-08-05 00:36:22.395626", 
 "modified_by": "Administrator", 
 "module": "Golden", 
 "name": "Stock Card by Location", 
 "owner": "Administrator", 
 "ref_doctype": "Stock Ledger Entry", 
 "report_name": "Stock Card by Location", 
 "report_type": "Script Report", 
 "roles": [
  {
   "role": "Stock User"
  }, 
  {
   "role": "Accounts Manager"
  }
 ]
}